dockstore-integration-testing/src/test/java/io/dockstore/client/cli/ToolsWorkflowTestIT.java

```diff
@@ -100,22 +100,21 @@ public void testWorkflowToolCWL() throws IOException, ApiException {
         // Repo: test_workflow_cwl
         // Branch: master
         // Test: normal cwl workflow DAG
-        // Return: JSON string with two tools, in grep-and-count.cwl and arguments.cwl
+        // Return: JSON string with three tools, in arguments.cwl, grep.cwl, and wc.cwl
 
         final List<String> strings = getJSON("DockstoreTestUser2/test_workflow_cwl", "/1st-workflow.cwl", "cwl", "master");
         int countNode = countToolInJSON(strings);
 
         Assert.assertTrue("JSON should not be blank", strings.size() > 0);
-        Assert.assertEquals("JSON should have two tools with docker images, has " + countNode, 2, countNode);
+        Assert.assertEquals("JSON should have three tools with docker images, has " + countNode, 3, countNode);
```

Comment:
Could just leave out "three", it's already specified in the other parameter. If the error message appears, it'll automatically mention 3 regardless